src/components/SoundToggle.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <p>Sound: <a href="#" v-on:click="onClick"><span v-if="sound">On</span><span v-else>Off</span></a></p>
```

Comment:
Can this have sound off/on icon instead? Like: https://material.io/icons/#ic_volume_off https://material.io/icons/#ic_volume_up